app/src/main/java/fr/free/nrw/commons/upload/worker/UploadWorker.kt

```diff
@@ -540,7 +541,11 @@ class UploadWorker(var appContext: Context, workerParams: WorkerParameters) :
         val intent = Intent(appContext,toClass)
         return TaskStackBuilder.create(appContext).run {
              addNextIntentWithParentStack(intent)
-             getPendingIntent(0, PendingIntent.FLAG_UPDATE_CURRENT)
+             if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
```

Comment:
While the mandate has been since Android S, can we not send the flag for `VERSION_CODES.M` and above since the IMMUTABLE flag is available since that version?